crates/amaru-ledger/Cargo.toml

```diff
@@ -22,6 +22,7 @@ thiserror.workspace = true
 iter-borrow.workspace = true
 amaru-kernel.workspace = true
 amaru-ouroboros-traits.workspace = true
+sha3 = "0.10.8"
```

Comment:
We want to assess this lib for security and all. Can't we find the necessary bits in one of the lib we already use?

Reply:
No, I could not find anything related to sha3 in any of our current dependencies. I'll double check though